it/uk/gov/hmrc/personaldetailsvalidation/PersonalDetailsValidationISpec.scala

```diff
@@ -44,6 +54,27 @@ class PersonalDetailsValidationISpec extends BaseIntegrationSpec {
       verifyMatchingStatusInAuditEvent(matchingStatus = "success")
     }
 
+    "return OK and insert records into both databases when provided personal details can be matched by Authenticator" in new Setup {
+      AuthenticatorStub.expecting(personalDetails).respondWithOK()
+      CitizenDetailsStub.expecting().respondWithOK()
+
+      val createResponse: WSResponse = sendCreateValidationResourceRequest(personalDetails).futureValue
+      await(associationRepository.insertRecord(association))
+      pdvRepository.create(personalDetailsValidation)
+
+      createResponse.status mustBe CREATED
+
+      val storedAssociation: Option[Association] = await(associationRepository.getRecord(encryptedCredID, encryptedSessionID))
```

Comment:
this will need to be wrapped in an eventually

Reply:
please address, its a fire and forget call, this will fail intermittently when attempting to read from db